src/ImageSharp/Formats/Ani/AniFrameMetadata.cs

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Six Labors.
+// Licensed under the Six Labors Split License.
+
+using SixLabors.ImageSharp.Metadata;
+using SixLabors.ImageSharp.PixelFormats;
+
+namespace SixLabors.ImageSharp.Formats.Ani;
+
+/// <summary>
+/// Provides Ani specific metadata information for the image.
+/// </summary>
+public class AniFrameMetadata : IFormatFrameMetadata<AniFrameMetadata>
+{
+    /// <summary>
+    /// Initializes a new instance of the <see cref="AniFrameMetadata"/> class.
+    /// </summary>
+    public AniFrameMetadata()
+    {
+    }
+
+    /// <summary>
+    /// Gets or sets the display time for this frame (in 1/60 seconds)
+    /// </summary>
+    public uint Rate { get; set; }
+
+    /// <summary>
+    /// Gets or sets the frames count of **one** "icon" chunk.
+    /// </summary>
+    public int FrameCount { get; set; } = 1;
```

Comment:
This should be the sequence number.

Reply:
It is actually intentionally designed as `FrameCount`, because each frame of `ani` file may contain multiple subframes (similar to different resolution ICOs), so this field is used to indicate how many subframes are contained in this frame

Reply:
Yes. The way I'm thinking is to use a `SequenceNumber` property to identify which individual frames of different resolutions belong to a single ANI frame. Imagine the individual frames following this sequence:
1,1,1,2,3,4,4,4. 

When we write the encoder (we'll need one to complete this PR) we can then use those sequence numbers to group the frames for encoding.